dotnet/src/Connectors/Connectors.UnitTests/Connectors.UnitTests.csproj

```diff
@@ -20,6 +20,10 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+
+    <!-- Because some of the referenced projects have dependencies that themselves have System.Text.Json set with a minimum of 7.0. -->
```

Comment:
In those cases can't you just let those references win?  Why do you need an additional reference here?

Reply:
Because otherwise it produces nuget conflicts warnings